butterknife-annotations/src/main/java/butterknife/Optional.java

```diff
@@ -0,0 +1,17 @@
+package butterknife;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.CLASS;
+
+/**
+ * Denote that the view specified by the injection is not required to be present.
+ * <pre><code>
+ * {@literal @}Optional @OnClick(R.id.subtitle) void onSubtitleClick;
```

Comment:
Oops, copy pasta mistake since [the original annotation had it](https://github.com/JakeWharton/butterknife/blob/9970b0a88a9f718f7036a71d86686432caea0f6e/butterknife/src/main/java/butterknife/Optional.java#L13)